Lib/inspect.py

```diff
@@ -1580,7 +1580,7 @@ def getattr_static(obj, attr, default=_sentinel):
         klass = type(obj)
         dict_attr = _shadowed_dict(klass)
         if (dict_attr is _sentinel or
-            isinstance(dict_attr, types.MemberDescriptorType)):
+            type(dict_attr) is types.MemberDescriptorType):
```

Comment:
It helps to comment on why we use type here.

Reply:
If it is required, every line in this function should have same comment.  It will be too noisy...

Reply:
Maybe a link to the issue?

Reply:
Note that I merged because I didn't read the comment / docstring so I didn't notice dynamic lookup should be avoided.  It's totally my mistake.

But note that It is not because I didn't notice isinstance cause dynamic lookup.
If we want to avoid this type of regression by adding comment, we need to add
`# to avoid dynamic lookup` comment on every line.  It's nonsense...